src/ovp_pipeline/commands/chat_handler.py

```
@@ -0,0 +1,794 @@
+    Writes ``50-Inbox/02-Tasks/ABSORB-chat-<id>-turn-<n>.md`` with
+    the assistant body of turn ``turn_number`` as content, plus
+    frontmatter pointing back at the originating chat.  Emits the
+    ``chat_writeback_handoff`` audit event.
+
+    Invariant: this is the **only** path from inquiry into
+    knowledge state, and it routes through the same review-gated
+    pipeline operator-written notes use.  Nothing in M21 writes
+    to ``objects`` / ``claims`` / ``evergreens`` directly.
+    """
+    if turn_number < 1:
+        raise ValueError(f"turn_number must be >= 1, got {turn_number}")
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)